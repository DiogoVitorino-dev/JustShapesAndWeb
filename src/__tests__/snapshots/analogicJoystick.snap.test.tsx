import { render } from "@testing-library/react-native";

import AnalogicJoystick from "@/controllers/mobile/analogic";

describe("Analogic Joystick - Snapshot mobile controller test", () => {
  it("should render correctly", () => {
    const tree = render(
      <AnalogicJoystick onMove={() => {}} size={300} velocity={50} />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
