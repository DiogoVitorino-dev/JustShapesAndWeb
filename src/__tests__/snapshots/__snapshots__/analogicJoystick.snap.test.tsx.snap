// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Analogic Joystick - Snapshot mobile controller test should render correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "indigo",
        "position": "absolute",
      },
      undefined,
    ]
  }
>
  <View
    collapsable={false}
    style={
      [
        {
          "borderRadius": 150,
          "height": 300,
          "width": 300,
        },
        {
          "alignItems": "center",
          "backgroundColor": "tomato",
          "justifyContent": "center",
        },
      ]
    }
    testID="pan"
  >
    <View
      animatedStyle={
        {
          "value": {
            "transform": [
              {
                "translateX": 0,
              },
              {
                "translateY": 0,
              },
            ],
          },
        }
      }
      collapsable={false}
      style={
        {
          "backgroundColor": "cyan",
          "borderRadius": 150,
          "height": 120,
          "transform": [
            {
              "translateX": 0,
            },
            {
              "translateY": 0,
            },
          ],
          "width": 120,
        }
      }
    />
  </View>
</View>
`;
